src/context/ArchiveContext/ArchiveContext.jsx

```diff
@@ -0,0 +1,71 @@
+import { useContext, createContext, useState, useEffect } from 'react';
+import { useAxios } from '../../CustomHooks';
+
+const ArchiveContext = createContext();
+
+const ArchiveProvider = ({ children }) => {
+	const [archiveNotes, setArchiveNotes] = useState([]);
+	const { response: archiveResponse, operation: fetchArchive } = useAxios();
+
+	useEffect(() => {
+		if (archiveResponse != null) {
+			if (archiveResponse.archives) {
+				setArchiveNotes(archiveResponse.archives);
+			}
+		} else {
+			getArchive();
+		}
+	}, [archiveResponse]);
+	const getArchive = () => {
+		fetchArchive({
+			method: 'GET',
+			url: '/api/archives',
+			headers: {
+				authorization:
+					'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI0OGJmMzFiYi1iYWQyLTQ3N2EtYmRhNy1kZDc4NDIxNDZhODgiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.kH6xXhMMvQ0sgsCz7e8vrSh1pVddplgC3IMzUSlTFsQ',
+			},
+		});
+	};
+	const notesToArchive = (noteId, notes) => {
+		fetchArchive({
+			method: 'POST',
+			url: `/api/notes/archives/${noteId}`,
+			data: {
+				notes,
+			},
+			headers: {
+				authorization: localStorage.getItem('token'),
+			},
+		});
+	};
+
+	const deleteArchiveNotes = (noteId) => {
+		fetchArchive({
+			method: 'DELETE',
```

Comment:
better alias of the operation function from useAxios can be used here.
Like `operationArchive`.
Reason: fetchArchive by name means a get request but you're doing `delete` here

Reply:
Check this in other contexts as well

Reply:
I completely agree with this point. Thanks for pointing it out. @sudipkundu999